mne/viz/utils.py

```diff
@@ -1297,3 +1297,80 @@ def _compute_scalings(scalings, inst):
         scale_factor = np.max(np.abs(scale_factor))
         scalings[key] = scale_factor
     return scalings
+
+
+class DraggableColorbar(object):
+    """Class for enabling interactive colorbar.
+    See http://www.ster.kuleuven.be/~pieterd/python/html/plotting/interactive_colorbar.html
```

Comment:
I don't see any license info there, can you check?

If there is no license, then by default it receives automatically a restrictive license and we cannot include it, so you'd have to contact the author.